<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Estadistics</class>
 <widget class="QDialog" name="Estadistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>statistics</string>
  </property>
  <widget class="QTableWidget" name="tableTeam1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>521</width>
     <height>201</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>511</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    background-color: #85C1E9; 
}
QTableWidget::item:selected {
    background-color: #5DADE2;
    color: white;
}</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>5</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Player Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Kills</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Deaths</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>+/-</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Points</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableTeam2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>521</width>
     <height>201</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    background-color: #F5CBA7;
}
QTableWidget::item:selected {
    background-color: #EB984E; 
    color: black;
}</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>5</number>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Player name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Kills</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Deaths</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>+/-</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Points</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="copaAzul">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="recursos.qrc">:/imagenes/imagenes/estadistics/azul.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="copaNaranja">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="recursos.qrc">:/imagenes/imagenes/estadistics/naranja.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="fondo">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>10</y>
     <width>721</width>
     <height>581</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="recursos.qrc">:/imagenes/imagenes/fondos/fondo4.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelWinner">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>fondo</zorder>
  <zorder>tableTeam1</zorder>
  <zorder>tableTeam2</zorder>
  <zorder>copaAzul</zorder>
  <zorder>copaNaranja</zorder>
  <zorder>labelWinner</zorder>
 </widget>
 <resources>
  <include location="Desktop/merge con server/counter-strike-2D/client/lobby/recursos.qrc"/>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
